infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -269,8 +269,9 @@ repositoryEnable: true
 # ===================
 
 prometheusEnable: true
-# prometheus:
-#    special configuration over the prometheus sub-chart
+kube-prometheus-stack:
+grafana:
```

Comment:
```suggestion
  grafana:
```

Reply:
You will need to add some space here @psschwei make it work I think

Reply:
:facepalm: